iceoryx2/src/service/port_factory/client.rs

```diff
@@ -77,6 +81,7 @@ pub struct PortFactoryClient<
     >,
     pub(crate) max_loaned_requests: usize,
     pub(crate) unable_to_deliver_strategy: UnableToDeliverStrategy,
+    pub(crate) degration_callback: Option<DegrationCallback<'static>>,
```

Comment:
```suggestion
    pub(crate) degradation_callback: Option<DegradationCallback<'static>>,
```

Similar in other places.